changelogs/fragments/vlans_tests_coverage_improvement.yaml

```diff
@@ -0,0 +1,3 @@
+---
+minor_changes:
```

Comment:
As you did only changes in tests this will be trivial change right?

Reply:
There is minor change in facts as well.